dali/operators/image/convolution/laplacian_params.h

```diff
@@ -0,0 +1,302 @@
+// Copyright (c) 2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_OPERATORS_IMAGE_CONVOLUTION_LAPLACIAN_PARAMS_H_
+#define DALI_OPERATORS_IMAGE_CONVOLUTION_LAPLACIAN_PARAMS_H_
+
+#include <numeric>
+#include <vector>
+
+#include "dali/core/small_vector.h"
+#include "dali/pipeline/data/views.h"
+#include "dali/pipeline/operator/common.h"
+
+
+namespace dali {
+namespace laplacian {
+
+constexpr static const char *kWindowSizeArgName = "window_size";
+constexpr static const char *scaleArgName = "scale";
+constexpr static const float scaleArgDefault = 1.0f;
+constexpr static const char *normalizeArgName = "normalize";
+constexpr static const bool normalizeArgDefault = false;
+
+// OpenCV uses windows sizes up to 31, but may be run on double precision floats.
```

Comment:
Can you explain why OpenCV is referenced here? Maybe you should add a note that this operators tries to mimic OpenCV behavior.

Reply:
It does. I'll expand the comment.

Reply:
done